File: API/CollectionAPI/Models/CollectionBill.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace CollectionAPI.Models;

public class CollectionBill
{
    public int BillId { get; set; }
    public double LabourCharges { get; set; }
    public int Amount { get; set; }
    public string? FarmerName { get; set; }
    public DateTime BillingDate { get; set; }
    public DateTime CollectionDate { get; set; }
    
}

File: UI/FrontEnd/src/app/vendors/collectionbill.ts
export class Collectionbill {
    constructor(
        public billId:number,
        public collectionId:number,
        public labourCharges:number,
        public amount:number,
        public farmerName:string,
        public billingDate:string,
        public collectionDate:string,
    ){}
}
